ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml

```diff
@@ -65,6 +65,18 @@ releases:
       include_built_images: true
       name: "4.14"
       namespace: ocp
+  openshift-4_13:
```

Comment:
@coreydaley please do let me know if you have any better suggestions for naming this.
It didn't allow me use `.` in name, so I used `_`

Reply:
May be it can be `openshift_4_13` 🤔